src/source/PeerConnection/SessionDescription.c

```diff
@@ -642,9 +642,8 @@ STATUS deserializeRtcIceCandidateInit(PCHAR pJson, UINT32 jsonLen, PRtcIceCandid
     tokenCount = jsmn_parse(&parser, pJson, jsonLen, tokens, ICE_CANDIDATE_INIT_TOKENS);
     CHK(tokenCount > 1, STATUS_INVALID_API_CALL_RETURN_JSON);
     CHK(tokens[0].type == JSMN_OBJECT, STATUS_ICE_CANDIDATE_INIT_MALFORMED);
-    CHK(tokenCount == ICE_CANDIDATE_INIT_TOKENS, STATUS_ICE_CANDIDATE_INIT_MALFORMED);
 
-    for (i = 1; i < tokenCount; i++) {
+    for (i = 1; i < (tokenCount - 1); i++) {
```

Comment:
Can we get more info on i < tokenCount

Reply:
When looping for key/value pairs it checks for `candidate`.

 It would incorrectly accept a key (without a value) at the end and then do an invalid read.

Reply:
I see, we normally bump up the value if i when "consuming" the value too.